pftools/python/requirements.txt

```diff
@@ -1,5 +1,8 @@
 PyYAML==6.0.1
+ecmwflibs
```

Comment:
I'm going to remove my additions here since you changed the reader.   I don't think we are using these readers.